analysis/source/prepare_data/SConscript

```diff
@@ -20,10 +20,9 @@ env.BuildPython(target  = ['#%s/data.txt'       % build],
 #                 source  = ['#%s/create_data.m'   % source],
 #                 log_ext =  'create_data')
 
-# # Not Windows
```

Comment:
Why'd we cut this? The shell script won't run on Windows.

Reply:
Ah I see, the builder works in Windows but not the script. Feel free to add back, my bad.